modelci/hub/converter.py

```diff
@@ -27,9 +27,10 @@
 from hummingbird.ml import constants as hb_constants
 from hummingbird.ml.convert import _convert_xgboost, _convert_lightgbm, _convert_onnxml, _convert_sklearn  # noqa
 from hummingbird.ml.operator_converters import constants as hb_op_constants
-from hummingbird.ml.supported import xgb_operator_list, lgbm_operator_list, sklearn_operator_list
+from lightgbm import LGBMModel
```

Comment:
Seems this is a new package.

Why not update the requirement.txt

Reply:
The requirement.txt already includes the packages: `lightbgm`, `py-xgboost` and `scikit-learn`